README.md

```diff
@@ -35,6 +35,9 @@ application with a Node.js backend and a MySQL database.
 application with a Rust backend and a Postgres database.
 - [`Spring / PostgreSQL`](https://github.com/docker/awesome-compose/tree/master/spring-postgres) - Sample Java application
 with Spring framework and a Postgres database.  
+- [`Phoenix(Elixir) / PostgreSQL / pgAdmin 4`](https://github.com/docker/awesome-compose/tree/master/phoenix-postgres-pgadmin) - Sample Elixir 
```

Comment:
Remove the reference to pgAdmin and rename the main directory

Reply:
done!